knowledge-base/grid-persist-selection-across-pages.md

```diff
@@ -20,3 +20,6 @@ How do I select data across different pages without losing the current selection
 If your users don't use the `Ctrl` and/or `Shift` buttons to extend the selection, you may want to implement some logic that amends the current selection. This also extends to the Select All checkbox in the header.
 
 An example is available in the following project: [https://github.com/telerik/blazor-ui/tree/master/grid/persist-selection](https://github.com/telerik/blazor-ui/tree/master/grid/persist-selection)
+
+>tip Since our 2.10.0 release the behavior showcased above is delivered out of the box with the `SelectAllMode` parameter of the `GridCheckBoxColumn`.
+More information can be found in the following link: https://docs.telerik.com/blazor-ui/components/grid/selection/multiple
```

Comment:
Use a slug here

Reply:
Also, add a similar note in the readme of the public project https://github.com/telerik/blazor-ui/tree/master/grid/persist-selection

Reply:
Done